test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] Kindly remove the `test.md` file because must not be presented in your repo as required (it must be ignored by git). :wink: